lib/plugins/aws/deploy/compile/functions/tests/index.js

```diff
@@ -114,12 +114,17 @@ describe('AwsCompileFunctions', () => {
         .IamPolicyLambdaExecution
         .Properties
         .PolicyName
-      ).to.equal(
-        `${
-          awsCompileFunctions.options.stage
-        }-${
-          awsCompileFunctions.serverless.service.service
-        }-lambda`
+      ).to.deep.equal(
+        {
+          'Fn:Join': [
+            '-',
+            [
+              '${opt:stage}',
+              '${self:service}',
```

Comment:
These variables will be rendered by @eahefnawy's code prior to deployment, right?